docs/tutorials/migrate-from-mbed-os-2-to-5.md

```diff
@@ -348,3 +348,13 @@ Now, the program successfully compiles.
 ### Runtime errors
 
 Although the program or library now compiles successfully, runtime errors may still be present. Please visit the [compile-time errors tutorial](compile-time-errors.html#runtime-errors-and-lights-of-the-dead) for further debugging tips about common errors.
+
+### Enabling Mbed OS bare metal
+
+Enabling the Mbed OS bare metal profile allows you to build Mbed OS without an RTOS. To enable it, you have to complete the [migration to Mbed OS 5](#migrating-to-mbed-os-5). Once the migration is complete, you can enable Mbed OS bare metal by creating an `mbed-app.json` with the following contents:
```

Comment:
@AnotherButler isn't this file called `mbed_app.json`? (with and underscore)

Reply:
I'll fix that

Reply:
hooray!